WordPress/Classes/ViewRelated/Blog/Blog Dashboard/DashboardCard.swift

```diff
@@ -21,6 +21,8 @@ enum DashboardCard: String, CaseIterable {
     /// Card placeholder for when loading data
     case ghost
     case failure
+    /// Empty state when no cards are present
+    case empty
```

Comment:
Not sure if that's ideal, but other state views such as `ghost` and `failure` are also displayed using `DashboardCard`.